aspnetcore/fundamentals/websockets.md

```diff
@@ -88,8 +88,16 @@ When accepting the WebSocket connection before beginning the loop, the middlewar
 
 Windows Server 2012 or later and Windows 8 or later with IIS/IIS Express 8 or later has support for the WebSocket protocol.
 
+> [!NOTE]
+> WebSockets are always enabled when using IIS Express.
```

Comment:
WebSockets are ~always~ enabled when using IIS Express.

Reply:
I like "always' here to make it clear nothing has to be done to enable them.

Reply:
Ok, then
WebSockets are always totally enabled when using IIS Express. :)

I think it's pretty clear without the dead wood.